### DISCUSSION TITLE
My feed broke

### DISCUSSION THREAD

**Discussion by Skiestrixx** [spacecowboy/Feeder#153] [Q&A] [answered]
I have an issue where it suddenly broke and stop giving me new results from [this site](https://www.sunstar.com.ph/api/v1/collections/home.rss), tested on other rss readers it works fine. 

App Version 2.6.11

![Screenshot_20231217_221141_Feeder.jpg](https://github.com/spacecowboy/Feeder/assets/90496866/5a63ffff-64e6-4485-a6d1-c41527980644)
![Screenshot_20231217_221207_Pluma.jpg](https://github.com/spacecowboy/Feeder/assets/90496866/b18eb059-b245-44a7-a9a1-90fafbf37518)

**Comment by spacecowboy** [comment]
The message says why. They are returning "null" as the content-type of the response which is just plain wrong.

email them and ask them to fix their webserver

**Reply by Skiestrixx** [reply]
Tested it on numerous rss readers which it was able to resolve how would that be a problem on their side. (genuine question)

**Reply by spacecowboy** [reply]
They ARE returning the wrong content type according to the HTTP spec.

Most feed readers just ignore it

**Reply by Skiestrixx** [reply]
Cool thanks for the reply I'll contact the site owner later. Just another quick question I have another feed that didn't load and the error message is invalid XML, I assume they aren't following the standard right? (New to RSS)

**Reply by spacecowboy** [reply, marked as answer]
Invalid XML means the file has an error in it. No parser will be able to read it.
